src/lib/ppx_coda/versioned.ml

```diff
@@ -39,6 +45,8 @@ open Ppxlib
 
 type version_info = {versioned: bool; wrapped: bool}
```

Comment:
I think this type isn't used any more, would be good to get rid of it.

Reply:
Good catch, made that change.